docs/openapi/components/schemas/common/ppq203.yml

```diff
@@ -131,16 +131,18 @@ example: |-
         "date": "2019-12-11T03:50:55Z",
         "property": {
           "type": [
-            "ChemicalProperty"
+            "MechanicalProperty"
```

Comment:
This will need a further update - e.g. should be more something like a contamination by white flies